framework/core/events_test.ts

```diff
@@ -0,0 +1,687 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
```

Comment:
This file was copied over from the Deno repo as is. I'm not sure if we need to include a copy of the MIT license verbiage here also.

Reply:
let's keep the license, thanks!